idea_town/experiments/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+from ..utils import HashedUploadTo
+
+
+experiment_thumbnail_upload_to = HashedUploadTo('thumbnail')
+experimentdetail_image_upload_to = HashedUploadTo('image')
+
+
+class Experiment(models.Model):
+
+    title = models.CharField(max_length=128)
+    slug = models.SlugField(max_length=128, unique=True, db_index=True)
+    thumbnail = models.ImageField(upload_to=experiment_thumbnail_upload_to)
+    description = models.TextField()
+
+    users = models.ManyToManyField(User, through='UserInstallation')
+
+    created = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.title
+
+
+class ExperimentDetail(models.Model):
```

Comment:
Actually, that's a question I had and I'd punted for now in the model.

We can (maybe should?) host the experiments, and can treat them as uploads like the images? Or, we might put them on AMO? Not sure what exactly we want to do there yet.